Bender.local

```diff
@@ -0,0 +1,6 @@
+# Copyright 2020 ETH Zurich and University of Bologna.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+overrides:
+    axi: {git: https://github.com/pulp-platform/axi.git, rev: 45cfe01040d3db5d4aece8cc0e439e4e4ff13672}
```

Comment:
Just out of curiosity; this overrides axi everywhere in the project?
That seems a bit taped together but okay sure

Reply:
Yes! Because the sub-dependent repos of Snitch uses different AXI versions. So ETHZ added a Bender.local to override the dependency issue. Without this Bender will always prompt which version to use.